test/attio/metadata/metadata.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/amp-labs/connectors/test/attio"
+)
+
+func main() {
+	ctx := context.Background()
+
+	conn := attio.GetAttioConnector(ctx)
+
+	m, err := conn.ListObjectMetadata(ctx, []string{"objects", "lists", "self", "workspace_members", "webhooks", "tasks"})
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I was told to use `slog` lib and avoid `log`.
There is `utils.Fail()` that does that underneath.